app/config/security.yml

```diff
@@ -34,7 +34,7 @@ security:
                 oauth_user_provider:
                     service: korobi.user_provider
                 success_handler: korobi.authentication.login.handler
-            login:
+            form_login:
                 success_handler: korobi.authentication.login.handler
             logout:
```

Comment:
Apparently, yeah. See http://symfony.com/doc/current/reference/configuration/security.html

I can't say it makes much sense to me, but the site still seems to work.